tests/components/geonetnz_volcano/test_init.py

```diff
@@ -0,0 +1,48 @@
+"""Define tests for the GeoNet NZ Volcano general setup."""
+from asynctest import CoroutineMock, patch
+import pytest
+
+from homeassistant.components.geonetnz_volcano import DOMAIN, FEED
+from homeassistant.const import (
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_RADIUS,
+    CONF_SCAN_INTERVAL,
+    CONF_UNIT_SYSTEM,
+)
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def config_entry():
```

Comment:
If this fixture is the same as the one in the config flow tests module we can make a common fixture in a `conftest.py` module in the geonetz_volcano test package.

Reply:
Thanks for the hint, I wasn't aware of the `conftest.py` magic.